### PR TITLE
[react-json-view] Add minified version of js

### PR DESCRIPTION
**Pull request by ignorabilis**
[react-json-view] 
Add minified version of js

### CODE REVIEW

File: react-json-view/boot-cljsjs-checksums.edn [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Deraen) [inline react-json-view/boot-cljsjs-checksums.edn]: This file is missing contents. Please run `boot package` and commit updated file.

Comment (ignorabilis) [reply react-json-view/boot-cljsjs-checksums.edn]: @Deraen - I ran `boot package` from Windows; while it worked I couldn't find a way to debug why checksums were left empty. Anyway, I generated these via a Linux Docker container so it should be ok now.

Comment (Deraen) [reply react-json-view/boot-cljsjs-checksums.edn]: Sorry about windows support. Probably this problem was caused by boot-cljsjs not taking Windows path delimiter into account in this regex which is used to select which files to add to checksum file: https://github.com/cljsjs/boot-cljsjs/blob/master/src/cljsjs/boot_cljsjs/packaging.clj#L281

I tried to fix this in new version but not sure if it works yet: https://github.com/cljsjs/boot-cljsjs/commit/e538c38992e2baeaf1d4c282edae63ff85a81899

Comment (ignorabilis) [reply react-json-view/boot-cljsjs-checksums.edn]: @Deraen - I just downloaded and tested, it's working now.

**Pull request merged**